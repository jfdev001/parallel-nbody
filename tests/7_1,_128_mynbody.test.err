Reservation number 3909823: Reserved 7 hosts for 900 seconds 
Run on 7 hosts for 960 seconds from Sun Jan 15 15:38:25 2023
: node058/0 node059/0 node060/0 node061/0 node062/0 node063/0 node064/0 
Running N-body with 128 bodies and 1000 steps
