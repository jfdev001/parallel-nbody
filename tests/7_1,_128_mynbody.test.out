[node058:5747] *** An error occurred in MPI_Gather
[node058:5747] *** reported by process [3755409409,0]
[node058:5747] *** on communicator MPI_COMM_WORLD
[node058:5747] *** MPI_ERR_TRUNCATE: message truncated
[node058:5747] *** MPI_ERRORS_ARE_FATAL (processes in this communicator will now abort,
[node058:5747] ***    and potentially your MPI job)
STATUS=PASSED
